general/about_ubuntu_touch.rst

```diff
@@ -0,0 +1,20 @@
+About Ubuntu Touch
```

Comment:
This is nice text, but it's in my opinion beyond the scope of the docs page. This would belong on ubunu-touch.io or ubports.com.

Reply:
It was written by Ralf_W.... so he can take it back to marketing. :P  I can write something smaller and more concise for the docs.